datafusion/proto-common/src/from_proto/mod.rs

```diff
@@ -1066,6 +1066,7 @@ impl TryFrom<&protobuf::TableParquetOptions> for TableParquetOptions {
                 .unwrap(),
             column_specific_options,
             key_value_metadata: Default::default(),
+            crypto: Default::default(),
```

Comment:
If we aren't going to support conversion to protobuf yet, we should probably raise an error in `impl TryFrom<&TableParquetOptions> for protobuf::TableParquetOptions` if any crypto options are set.

Reply:
Should we? I see that `key_value_metadata` is always set to empty here despite being defined in datafusion/proto-common/src/generated/prost.rs. So, it's not clear to me that we should throw an error here.